php_phongo.c

```diff
@@ -2445,6 +2445,12 @@ PHP_MINIT_FUNCTION(mongodb)
 
 	/* Initialize libmongoc */
 	mongoc_init();
+
+#ifdef MONGOC_EXPERIMENTAL_FEATURES
```

Comment:
Yes, Decimal128 is behind `BSON_EXPERIMENTAL_FEATURES`.

1.5 is expected to be released around October I imagine, week before MongoDB 3.4 as usual.

1.4 meanwhile, is a huge release, with incredible amount of work dedicated for the PHP drivers and TLS improvements. We had no reason to delay this release another 3months, as jam packed as it already is.

If you want to skip this release, then that is completely up to you. It might proof problematic for distros as there is no libmongoc-priv anymore, so you'd have to change the requirements to max 1.3. Other then then, I can't think of anything bad happening.